Author: Andreas Bombe <aeb@debian.org>
Subject: Add missing stdlib.h and ctype.h includes

src/mediactl.c needs ctype.h for its use of isspace().

src/v4l2subdev.c needs ctype.h for of isspace() and isupper() and
stdlib.h for strtoul().
    
Forwarded: yes
diff --git a/src/mediactl.c b/src/mediactl.c
index 46562de..c2f985a 100644
--- a/src/mediactl.c
+++ b/src/mediactl.c
@@ -29,6 +29,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <ctype.h>
 #include <string.h>
 #include <fcntl.h>
 #include <errno.h>
diff --git a/src/v4l2subdev.c b/src/v4l2subdev.c
index d0c37f3..87d7eb7 100644
--- a/src/v4l2subdev.c
+++ b/src/v4l2subdev.c
@@ -26,6 +26,8 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <stdbool.h>
+#include <stdlib.h>
+#include <ctype.h>
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
